README.md

```diff
@@ -1 +1,47 @@
-# ordinal-scale-stats-py
\ No newline at end of file
+# Ordinal-Scale-Stats-py
+
+Python package that helps you analyze ordinal data.
+
+## Introduction
+
+Ordinal scale data is common. Companies and governments can quickly perform large-scale research with surveys.
+Usually, a survey output is placed on the Likert scale, where answers are ordered to describe person's feelings about survey's topic.
+Typical example of a survey is when person is asked to agree with a statement with answers on a five-level scale:
+
+```text
+Should your personal data be protected by the law?
+
+1. Strongly disagree.
+2. Rather disagree.
+3. I don't know.
+4. Rather agree.
+5. Strongly agree.
+
+
+```
+
+The order between categories makes analysis complex, and the fact that answers are polarized between opposing states. Moreover, border between categories is subjective and it depends from the person's experiences, feelings, and knowledge about a surveying topic.
+
+Classical measurements of central tendency do not fit well ordinal data [ADD BIBLIOGRAPHY]. That's why we encourage you to use `ordinal-scale-stats` package for your analysis of survey responses.
```

Comment:
Also, "That's why we encourage you to use THE `ordinal-scale-stats` package for your analysis of survey responses."